### PR TITLE
{tools}[GCCcore/12.3.0] Rasqal v0.9.33, gtk-doc v1.34.0, Pygments v2.18.0, ...

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

resolves https://github.com/vscentrum/vsc-software-stack/issues/309

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@PetrKralCZ Some missing (build) dependencies for `gtk-doc`?
```
Found pkg-config: /usr/bin/pkg-config (1.4.2)
Did not find CMake 'cmake'
Found CMake: NO
Run-time dependency glib-2.0 found: NO (tried pkgconfig)

../gtk-doc-1.34.0/tests/meson.build:31:11: ERROR: Dependency "glib-2.0" not found, tried pkgconfig
```

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/r/Raptor/Raptor-2.0.16-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://download.librdf.org/source']
+sources = ['raptor2-%(version)s.tar.gz']
+checksums = ['089db78d7ac982354bdbf39d973baf09581e6904ac4c92a98c5caadb3de44680']
+
+builddependencies = [
+    ('Autotools',  '20220317'),
+    ('binutils',   '2.40'),
```
Comment (boegel) [inline easybuild/easyconfigs/r/Raptor/Raptor-2.0.16-GCCcore-12.3.0.eb:30]: Existing easyconfig for `Raptor` has a recent version of `make` as build dependency, we should keep it (it's likely that older versions of `make` won't work, and `make-4.4.1-GCCcore-12.3.0.eb` is available already anyway)

```suggestion
    ('binutils',   '2.40'),
    ('make',   '4.4.1'),

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/7d038c92b0a2f9adbc79007ad33b6e7e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**